### DISCUSSION TITLE
RabbitMQ Super Stream and C# Client

### DISCUSSION THREAD

**Discussion by PTArch** [rabbitmq/rabbitmq-dotnet-client#1275] [Q&A]
Hi, Is the last version of C# Client compatible with the super streams feature? Thanks.

**Comment by lukebakken** [comment]
Please see the https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/ project

cc @Gsantomaggio

**Comment by Gsantomaggio** [comment]
All the AMQP clients are compatible with the `stream queues`

The dedicated clients are for the `stream plugin`

Please see this[ link to see the differences](https://www.rabbitmq.com/docs/stream-core-plugin-comparison)
